hydrolib/core/basemodel.py

```diff
@@ -403,6 +404,29 @@ def pop_last_parent(self) -> None:
             self._anchors.pop()
 
 
+class CachedPathFileModelData:
```

Comment:
This name confuses me. Maybe just use `CachedFileModel`.

Reply:
will do